packages/manager/apps/cloud/client/app/common/translations/Messages_de_DE.json

```diff
@@ -108,9 +108,6 @@
   "cloud_menu_compute": "Infrastruktur",
   "cloud_menu_storage": "Storage",
   "cloud_menu_project_num": "Projekt {{num}}",
-  "dbaasts_menu_title": "DBaaS Time Series",
-  "dbaasts_menu_add": "Ein neues Projekt erstellen",
-  "dbaasts_menu_project_num": "Projekt {{num}}",
```

Comment:
Removed in `fr_FR`: https://github.com/ovh/manager/commit/be928ffc301f8f1c29d4afa15c2673d3816a1ee7#diff-60fd86fcaa5efc26ec8977b6867616bf